src/main/java/org/vaadin/easyuploads/UploadField.java

```diff
@@ -79,7 +79,7 @@
     private Upload upload;
     protected Component display;
 
-    private ProgressIndicator progress = new ProgressIndicator();
+    private ProgressBar progress = new ProgressBar();
```

Comment:
@mstahv we need to mention in the documentation that setting `getUI().setPollInterval()` may be needed or need to support it internally somehow.
Currently the progress bar is not updated by default.